packages/core/lib/Worker.js

```diff
@@ -59,59 +61,34 @@ function retrievePath(str) {
 }
 
 function getModule(mod) {
-  return mod.hasOwnProperty('default') ? mod.default : mod;
+  return Boolean(mod.default) ? mod.default : mod;
 }
 
-function setup(entryPath, babel) {
-  if (babel === 'babel') {
-    const presets = [];
-    if (presetEnv) {
-      presets.push([presetEnv.default, { targets: { node: true } }]);
-    }
-
-    presets.push(require('@babel/preset-typescript').default);
-
-    require('@babel/register')({
-      configFile: false,
-      babelrc: false,
-      presets,
-      plugins: [
-        require('@babel/plugin-proposal-class-properties').default,
-        require('@babel/plugin-proposal-nullish-coalescing-operator').default,
-        require('@babel/plugin-proposal-optional-chaining').default,
-        require('@babel/plugin-transform-modules-commonjs').default,
-      ],
-      extensions: [...DEFAULT_EXTENSIONS, '.ts', '.tsx'],
-      // By default, babel register only compiles things inside the current working directory.
-      // https://github.com/babel/babel/blob/2a4f16236656178e84b05b8915aab9261c55782c/packages/babel-register/src/node.js#L140-L157
-      ignore: [
-        // Ignore parser related files
-        /@babel\/parser/,
-        /\/flow-parser\//,
-        /\/recast\//,
-        /\/ast-types\//,
-      ],
-    });
-  }
+async function getModuleName(path) {
+  const moduleName = retrievePath(path).split('node_modules/')[1];
```

Comment:
! this is a potential bug, local codemods are normally not found in the node_modules dir....